hardening/anaconda/test.py

```diff
@@ -33,13 +34,16 @@
 g.install(kickstart=ks)
 
 with g.booted():
+    # copy our datastream to the guest
+    g.copy_to(util.get_datastream(), 'contest-ds.xml')
```

Comment:
I'd prefer keeping filename the same, `ssg-rhel{versions.rhel.major}-ds.xml`. Because I'm used to that. But it's not blocker, just my 2cents.

Reply:
Applies also to `oscap` test.

Reply:
Okay, make sense.